packages/cron/package.json

```diff
@@ -11,29 +11,170 @@
+    ],
+    "extends": [
+      "eslint:recommended",
+      "plugin:@typescript-eslint/recommended",
+      "plugin:@typescript-eslint/recommended-requiring-type-checking",
+      "standard",
+      "plugin:prettier/recommended"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+      "ecmaVersion": 13,
+      "project": [
+        "./tsconfig.json"
+      ]
+    },
+    "env": {
+      "es2022": true,
+      "browser": true,
+      "node": true
+    },
+    "rules": {},
+    "overrides": [
+      {
+        "files": [
+          "./src/bin/pins-failed.js",
+          "./src/jobs/dagcargo.js"
+        ],
+        "rules": {
+          "@typescript-eslint/no-unsafe-assignment": "off"
+        }
+      },
+      {
+        "files": [
+          "./src/jobs/dagcargo.js"
+        ],
+        "rules": {
+          "camelcase": "off"
+        }
+      },
+      {
+        "files": [
+          "./src/jobs/dagcargo.js",
+          "./src/jobs/metrics.js",
+          "./src/jobs/pins.js",
+          "./src/lib/ipfs.js"
+        ],
+        "rules": {
+          "@typescript-eslint/no-unsafe-member-access": "off"
+        }
+      },
+      {
+        "files": [
+          "./src/lib/ipfs.js"
+        ],
+        "rules": {
+          "@typescript-eslint/no-unsafe-argument": "off",
+          "@typescript-eslint/no-unsafe-assignment": "off"
+        }
+      },
+      {
+        "files": [
+          "./src/lib/fetch.js"
+        ],
+        "rules": {
+          "@typescript-eslint/no-unsafe-assignment": "off"
+        }
+      },
+      {
+        "files": [
+          "./src/jobs/dagcargo.js",
+          "./src/jobs/pinata.js",
+          "./src/jobs/pins.js",
+          "./src/lib/ipfs.js"
+        ],
+        "rules": {
+          "@typescript-eslint/restrict-template-expressions": "off"
+        }
+      },
+      {
+        "files": [
+          "./src/jobs/dagcargo.js",
+          "./src/jobs/pins.js",
+          "./src/lib/fetch.js",
+          "./src/lib/pinata.js"
+        ],
+        "rules": {
+          "@typescript-eslint/no-unsafe-return": "off"
+        }
+      },
+      {
+        "files": [
+          "./src/bin/dagcargo-sizes.js",
+          "./src/bin/metrics.js",
+          "./src/bin/pinata.js",
+          "./src/bin/pins-failed.js",
+          "./src/bin/pins.js"
+        ],
+        "rules": {
+          "@typescript-eslint/no-floating-promises": "off"
+        }
+      },
+      {
+        "files": [
+          "./src/bin/dagcargo-sizes.js",
+          "./src/bin/metrics.js",
+          "./src/bin/pinata.js",
+          "./src/bin/pins-failed.js",
+          "./src/bin/pins.js",
+          "./src/jobs/dagcargo.js",
+          "./src/jobs/pinata.js",
+          "./src/jobs/pins.js",
+          "./src/lib/pinata.js"
+        ],
+        "rules": {
+          "@typescript-eslint/naming-convention": "off"
+        }
+      }
+    ]
+  },
+  "eslintIgnore": [
+    "node_modules"
+  ]
```

Comment:
Please can you follow up with a PR to fix the linter issues and remove all this configuration? 🙏🙏🙏

Reply:
sure thing. didn't want to lint those old files prematurely until you weighed in on the lint rules at hand

Reply:
@alanshaw this would do it. Not sure if you want to review in here or out. https://github.com/gobengo/nft.storage/pull/1